py-polars/polars/internals/dataframe/frame.py

```diff
@@ -6668,23 +6669,17 @@ def rows(self, named: bool = False) -> list[tuple[Any, ...]] | list[Any]:
         ... )
         >>> df.rows()
         [(1, 2), (3, 4), (5, 6)]
-        >>> df.rows(named=True)
-        [Row(a=1, b=2), Row(a=3, b=4), Row(a=5, b=6)]
+        >>> df.rows(named=True)  # doctest: +IGNORE_RESULT
+        [{'a': 1, 'b': 2), {'a': 3, 'b': 4), {'a': 5, 'b': 6}]
```

Comment:
`doctest` seems to have trouble comparing lists of dicts, hence the `IGNORE_RESULT` here.

The error message:
```
Failed example:
    df.rows(named=True)
Expected:
    [{'a': 1, 'b': 2), {'a': 3, 'b': 4), {'a': 5, 'b': 6}]
Got:
    [{'a': 1, 'b': 2}, {'a': 3, 'b': 4}, {'a': 5, 'b': 6}]
```

Reply:
Oof, thanks!